packages/mongo-nav/src/user-menu/UserMenu.tsx

```diff
@@ -260,8 +260,10 @@ function UserMenu({
     },
   };
 
-  const urls: URLSInterface = defaultsDeep(urlsProp, defaultURLs);
+  const urls: Required<URLSDefaults> = defaultsDeep(urlsProp, defaultURLs);
   const userMenu = urls.userMenu ?? {};
+  const cloudUrls: Partial<UserMenuURLS['userMenu']['cloud']> =
+    userMenu.cloud ?? {};
```

Comment:
Wont userMenu.cloud always be defined now?

Reply:
We can't assume that from the outset, no

Reply:
Looking at the implementation on line `:263`, it looks otherwise. defaultsDeep ensures that every value in the urls object is defined, doesn't it? I might be missing something here...

Reply:
UserMenu should be safe to use outside of the Mongonav context

Reply:
Aren't we applying the defaults either way? Line 263 appears to apply default values regardless of context.

Reply:
yep you're right

Reply:
fixed